src/deployment/fabricProvisioningHelpers.ts

```diff
@@ -88,7 +88,7 @@ export async function initializeFabricProvisioningState(
         TelemetryActions.StartFabricProvisioningDeployment,
         {},
         {
-            localContainersInitTimeInMs: Date.now() - startTime,
+            fabricProvisioningInitTimeInMs: Date.now() - startTime,

```

Comment:
Why don't you use `startActivity` From telemetry? You can then use `end` or `endFailed` to send the final activity status. It does auto time calculation and keep it consistent with other activity based telemetry.